addons/stock_account/stock.py

```diff
@@ -160,6 +160,7 @@ def _get_invoice_line_vals(self, cr, uid, move, partner, inv_type, context=None)
             'price_unit': self._get_price_unit_invoice(cr, uid, move, inv_type),
             'discount': 0.0,
             'account_analytic_id': False,
+            'move_id': move.id if move.company_id.anglo_saxon_accounting else False,
```

Comment:
do we need to check move.company_id.anglo_saxon_accounting or can't we always pass move.id?